src/functions/template-tags/general.php

```diff
@@ -1380,11 +1380,15 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 			$excerpt = wp_trim_words( $excerpt, $excerpt_length, $excerpt_more );
 		}
 
-		// Remove all shortcode Content before removing HTML
-		if ( ! $allow_shortcode ) {
-			$excerpt = preg_replace( '#\[.+\]#U', '', $excerpt );
+		// If shortcode filter is enabled lets process them
+		if ( $allow_shortcode ) {
+			$excerpt = do_shortcode( $excerpt );
 		}
 
```

Comment:
> do we want to change it so we always process now?

We probably don't want to always process - but what I'm saying is if we _do_ process, we should probably also strip out any unprocessed shortcodes that remain, for the reasons described in my previous comment.

That secondary step could itself be turned on and off with a filter, to cover any unusual situations where cleanup isn't needed or desired.